rasa_core/policies/fallback.py

```diff
@@ -92,10 +92,8 @@ def predict_action_probabilities(self, tracker, domain):
         # it is set to 1.0 here in order
         # to not override standard behaviour
         nlu_confidence = nlu_data["intent"].get("confidence", 1.0)
-        if tracker.latest_action_name == self.fallback_action_name:
```

Comment:
I thought about it, let's keep this. If the user creates his custom fallback action and doesn't do the user utterance reverted event, it would be good to keep this.